shpc/main/modules/templates/docs.md

```diff
@@ -27,7 +27,7 @@ $ shpc install {{ name }}
 Or a specific version:
 
 ```bash
-$ shpc install {{ name }}:{{ versions.0 }}
```

Comment:
Ah! So it looks like we pass multiple attributes for config here (e.g., this example version, name and description)! Akin to how I did this for the other modules templates, let's clean up this render by just passing the config (and using it in that context in the templates). E.g.:

 - description -> config.description
 - name -> config.name
 - example_version -> config.tag.name
 - latest -> config.latest.name
etc.

And if you want we could change the current config (json dump of it) to config_json or something similar.

Reply:
Really good idea !